mods/d2k/chrome/ingame-player.yaml

```diff
@@ -278,7 +278,7 @@ Container@PLAYER_WIDGETS:
 							Children:
 								Image@ICON:
 									X: 5
-									Y: 4
+									Y: 5
 									ImageCollection: production-icons
 						ProductionTypeButton@VEHICLE:
 							Logic: AddRaceSuffixLogic
```

Comment:
I just noticed that these are in fact redundant now that we only have one artwork for all. Will fix it.